app/javascript/store/modules/froggo_team/index.js

```diff
@@ -0,0 +1,5 @@
+import actions from './actions'
```

Comment:
Definition for rule 'vue/max-len' was not found  vue/max-len<br>Missing semicolon                                semi